lib/pg_repack.sql.in

```diff
@@ -26,7 +26,7 @@ LANGUAGE sql STABLE STRICT SET search_path to 'pg_catalog';
 
 CREATE FUNCTION repack.get_index_columns(oid, text) RETURNS text AS
 $$
-  SELECT coalesce(string_agg(quote_ident(attname), $2), '')
+  SELECT coalesce(string_agg(quote_literal(attname), $2), '')
```

Comment:
This confused me a little bit and I had to follow the whole flow, using names requiring quoting, to convince myself it works as expected. This function output changed from:

```
create table "schema.x"."table.x" ("id.x" int, "id.y" int, data text, primary key ("id.x", "id.y"));

select repack.get_index_columns('"schema.x"."table.x_pkey"'::regclass::oid, ', ');
┌─[ RECORD 1 ]──────┬────────────────┐
│ get_index_columns │ "id.x", "id.y" │
└───────────────────┴────────────────┘
```
to 

```
piro=# select repack.get_index_columns('"schema.x"."table.x_pkey"'::regclass::oid, ', ');
┌─[ RECORD 1 ]──────┬────────────────┐
│ get_index_columns │ 'id.x', 'id.y' │
└───────────────────┴────────────────┘
```

But this is ok, because the escaping as identifier is now performed in the C `repack_trigger()`.

So this is alright but could you please add a comment? Something like "we are intentionally returning these columns as literal strings because they will be passed to a function which will quote them as identifiers"

Reply:
Sure, I'll add a comment around this code. Agree that this part is a bit confusing since it is later also quoted as identifiers and we cannot use `quote_ident` here neither.
I was thinking to move logic of `repack.get_index_columns` completely into C code in `repack.repack_trigger`. But I kept the changes a bit simpler at least at the first iteration. On the other hand if the whole logic will be in one place it might have better readability and I can move it into C if no objections.